src/d3a/d3a_core/sim_results/area_statistics.py

```diff
@@ -285,6 +315,19 @@ def _generate_intraarea_consumption_entries(accumulated_trades):
     return consumption_rows
 
 
+def generate_inter_area_trade_details(area, past_market_types):
+    accumulated_trades = _accumulate_grid_trades_all_devices(area, {}, past_market_types)
+    trade_details = list()
```

Comment:
Is it necessary to be a list of tuples ? It would be much cleaner to my eyes if it was a dictionary